js/webpack.config.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
         loader: 'file-loader'
       },
       {
-        test: /\.ico$/,
+        test: /parity-logo-black-no-text\.png$/,
```

Comment:
The reason for this one - we want the filename preserved in the copy. (Otherwise it ends up using the MD5 value as the filename and we cannot pull it in from the HTML)